Documentation/configuration.md

```diff
@@ -130,6 +130,10 @@ To start etcd automatically using custom settings at startup in Linux, using a [
 + default: none
 + env variable: ETCD_DISCOVERY_PROXY
 
+##### -strict-reconfig-check
++ Reject reconfiguration requests if the requests can cause quorum loss.
++ default: false
```

Comment:
I think it isn't true by default just for keeping existing behavior. If members don't have wrong peer URLs, adding multiple members isn't harmful (of course adding multiple members at once is not a best practce).